airflow/models/serialized_dag.py

```diff
@@ -142,23 +142,21 @@ def write_dag(
         # If Yes, does nothing
         # If No or the DAG does not exists, updates / writes Serialized DAG to DB
         if min_update_interval is not None:
-            if (
-                session.query(literal(True))
-                .filter(
+            if session.scalar(
+                select(literal(True)).where(
                     and_(
                         cls.dag_id == dag.dag_id,
                         (timezone.utcnow() - timedelta(seconds=min_update_interval)) < cls.last_updated,
                     )
```

Comment:
```suggestion
                    cls.dag_id == dag.dag_id,
                    (timezone.utcnow() - timedelta(seconds=min_update_interval)) < cls.last_updated,
```

This `and_` is totally unnecessary (`where` arguements are `and_`-ed by default), let’s use this change to get rid of it.

Reply:
done